nyudlts/ultraviolet
# Invenio114 deposit dates text - vanillajs

[Jira ticket invenio-114](https://jira.nyu.edu/browse/INVENIO-114)

This PR will add the following:
- deposit form > Recommended Information > Dates: a vanilla JS solution that changes the title from "Dates" to "Additional Dates"

To test this PR:
- `invenio-cli services start`
- `invenio-cli packages install site` (runs webpack build too)
- `invenio-cli run`
- [navigate to deposit](https://127.0.0.1:5000/uploads/new) > login > recommended information: observe date addition subtitle says "Additional Dates" and not "Dates"
  - checkout `dismorfo/ultraviolet-nightwatch` branch `invenio114-dates-text` and run the instructions for [PR3](https://github.com/dismorfo/ultraviolet-nightwatch/pull/3)

Thank you for your review.
